spec/features/onboarding_spec.rb

```diff
@@ -63,4 +63,21 @@
       expect(user.permissions).to eq User::Permissions::INSTRUCTOR
     end
   end
+
+  describe 'onboarding supplement' do
+    let(:onboarded) { false }
+
+    before :each do
+      login_as(user, scope: :user)
+    end
+
+    it 'should go to supplementary' do
+      visit onboarding_path
+      find('.intro .button').click
+      find('input[type=radio][value=true]').click
+      find('form button[type=submit]').click
+      sleep 1
+      expect(page).to have_content 'us?'
```

Comment:
The `sleep 1` is almost certainly not necessary. `have_content` will wait for a while until it matches.

This is not a very good way to test this. The language may change again the in the future, and this is a small enough snippet that it might match accidentally even if this feature was not working as intended.

You should instead do something like interact with the form, which should have a specific selector (such as an ID on the input) that is unique to this component.

Reply:
So, I can be certain that after `find('form button[type=submit]').click` is done, the next page will be loaded alright right? Also, I can just test if a specific component is present in the page right?